lib/index.js

```diff
@@ -31,15 +31,15 @@ internals.implementation = function (server, options) {
       if (!token && request.auth.mode === 'optional') {
         return reply.continue({ credentials: {} });
       }
-      
+
       if (!token) {
-        return reply(Boom.unauthorized('Missing auth token'));
+        return reply(Boom.unauthorized(null, 'Token'));
```

Comment:
Hello! I'm also interested in seeing this particular line merged.

I have two auth strategies in place for several routes, although the second strategy does little more than change the value of `request.auth.credentials` at this point :)

In any case, I did test the difference between `try` and `required` and did not see a difference in behavior. If the first argument to Boom.unauthorized is not null, then Hapi doesn't try other authentication strategies, regardless of `mode` selected.

Thanks!